app/models/service_plan.rb

```diff
@@ -12,4 +13,25 @@ class ServicePlan < ApplicationRecord
   has_many   :service_instances
 
   acts_as_taggable
+
+  def order(additional_parameters)
+    parsed_response = service_catalog_client.order_service_plan(name, service_offering.name, additional_parameters)
+
+    task = Task.create!(
+      :tenant => tenant,
+      :context => {
+        :service_instance => {
+          :source_id  => source.id,
+          :source_ref => parsed_response['metadata']['selfLink']
```

Comment:
We can do this in a follow-up but I'd prefer that we return this to the caller instead of the caller having to know the structure of the kubernetes response.